packages/react-mutation-mapper/src/component/column/Annotation.tsx

```diff
@@ -203,12 +213,19 @@ export function getAnnotationData(
                     resolveTumorType,
                     resolveEntrezGeneId
                 );
+                oncoKbAvailableDataTypes = _.uniq([
```

Comment:
seems like we're calling uniq twice once inside of calculateOncoKBAlavailabe... function ?  maybe just move this above if statement and allow it to handle empty onckbData.result?

Reply:
We need to make sure OncoKbCardDataType.BIOLOGICAL, 
 is always attached whether the if statement is passed or `_.values(oncoKbData.result.indicatorMap)` is not empty. Keeping it out of if statement would just make the logic more complex. We are scarifying the performance a bit here.